include/deal.II/fe/mapping_q_eulerian.h

```diff
@@ -200,6 +201,14 @@ class MappingQEulerian : public MappingQ<dim, spacedim>
     std::vector<Point<spacedim> >
     compute_mapping_support_points(const typename Triangulation<dim,spacedim>::cell_iterator &cell) const;
 
+    /**
+     * Always returns @p false because MappingQ1Eulerian does not in general
+     * preserve vertex locations (unless the translation vector happens to
+     * provide for zero displacements at vertex locations).
+     */
+    virtual
+    bool preserves_vertex_locations () const;
```

Comment:
Oh, I see. I had actually expanded all of the lines in between and not seen a class end `};`. It is a member class :)